contracts/ReputationMiningCycle.sol

```diff
@@ -8,9 +25,13 @@ import "./EtherRouter.sol";
 import "./ERC20Extended.sol";
 import "./ColonyNetworkStorage.sol";
 import "./IColonyNetwork.sol";
+import "./PatriciaTree/PatriciaTreeProofs.sol";
```

Comment:
None of these imports is used besides the last one. Not sure why `solium` doesn't error for this